### PR TITLE
chore: Update stableSwap APR for info page

### PR DESCRIPTION
**Pull request by Chef-Yogi**

### CODE REVIEW

**Comment by chefilip** [comment]
Screen Shot 2022-12-20 at 16 20 48
Could you help me update the logic in `src/lpArps` as well ? Thanks

**Comment by Chef-Yogi** [comment]
sure

**Comment by Chef-Yogi** [comment]
@chefilip Done

File: apis/farms/src/lpApr.ts [hunk 1]
```diff
@@ -153,13 +153,13 @@ const getAprsForStableFarm = async (stableFarm: any): Promise<BigNumber> => {
...
+    const day7Ago = sub(new Date(), { days: 7 })
 
-    const dayAgoTimestamp = getUnixTime(dayAgo)
+    const day7AgoTimestamp = getUnixTime(day7Ago)
 
-    const blockDayAgo = await getBlockAtTimestamp(dayAgoTimestamp)
+    const blockDay7Ago = await getBlockAtTimestamp(day7AgoTimestamp)
 
-    const { virtualPriceAtLatestBlock, virtualPriceOneDayAgo } = await stableSwapClient.request(
+    const { virtualPriceAtLatestBlock, virtualPriceOneDayAgo: virtualPrice7DayAgo } = await stableSwapClient.request(
```
Comment (chefjackson) [inline apis/farms/src/lpApr.ts:162]: Can we move this part to some kind of stable swap util file? So we can share code and don't need to implement the same logic in different places

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**